src/commands.ts

```diff
@@ -38,16 +36,23 @@ async function install ( file: vscode.Uri ) {
 
         term.dispose ();
 
+      } else if ( data.includes( `Use '--force'` ) ) {
+        Messages.retry ( file );
+
+        term.dispose ();
       } else if ( data.includes ( 'Installation ended' ) && !data.includes ( 'echo' ) ) {
 
         Messages.error ();
 
       }
 
     });
-
   }
 
+  term.sendText ( `${command} --install-extension "${file.fsPath}" ${force ? "--force" : ""}` );
+  term.sendText ( `echo 'Installation ended'` );
+
+  term.show ( false );
```

Comment:
Why did you move this lines to the bottom? Put them back in place.

Reply:
It is for subscribe to terminal before send text to it.

Reply:
It shouldn't make any difference as before the command gets executed we are already listening, but for better clarity I guess it makes sense to move them to the bottom, so let's keep them there, sorry about that.